### PR TITLE
Update protecting-proprietary-data section per customer feedback

### PR DESCRIPTION
**Pull request by antonfrMSFT**
Customer provided feedback that this section is misleading. 
Making modification to reflect that export settings and other changes don't affect user having access to dataset data.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@antonfrMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/visuals/power-bi-visualization-export-data.md [hunk 1]
```diff
@@ -149,24 +149,26 @@ When you select **Export**, Power BI exports the data to an *.xlsx* file and you
 
     - Since we applied filters to the visualization, the exported data will export as filtered. Notice that the first row displays **Applied filters: City is not Atlanta, GA**. 
 
-## Protecting proprietary data
+## Customize export data user expirience
```
Comment (mihart) [inline powerbi-docs/visuals/power-bi-visualization-export-data.md:152]: Customize the export data user experience

File: powerbi-docs/visuals/power-bi-visualization-export-data.md [hunk 2]
```diff
@@ -149,24 +149,26 @@ When you select **Export**, Power BI exports the data to an *.xlsx* file and you
...
 
-Your dataset may have content that should not be seen by all users. If you are not careful, exporting underlying data may let users see all the detailed data for that visual -- every column and every row in the data. 
+Users that granted access to report are **granted access to the entire dataset** unless [row-level security (RLS)](../admin/service-admin-rls.md) limits their access. Report authors and Power BI adminstrators may use capabilities below to customize user experiences when viewing or interacting or export data from reports:
```
Comment (mihart) [inline powerbi-docs/visuals/power-bi-visualization-export-data.md:154]: Users who are granted access to a report are **granted access to the entire underlying dataset**, unless [row-level security (RLS)(../admin/service-admin-rls.md) limits their access. Report authors and Power BI administrators can use the capabilities described below to customize the user experience.

File: powerbi-docs/visuals/power-bi-visualization-export-data.md [hunk 3]
```diff
@@ -149,24 +149,26 @@ When you select **Export**, Power BI exports the data to an *.xlsx* file and you
...
 
-- Power BI administrators can add [sensitivity labels](../admin/service-security-data-protection-overview.md) to dashboards, reports, datasets, and dataflows. They can then enforce protection settings such as encryption or watermarks when exporting data. 
+## Protect data when it is exported out of Power BI
 
-- Power BI administrators can use [Microsoft Cloud App Security](../admin/service-security-data-protection-overview.md) to monitor user access and activity, perform real-time risk analysis, and set label-specific controls. For example, organizations can use Microsoft Cloud App Security to configure a policy that prevents users from downloading sensitive data from Power BI to unmanaged devices. 
+- Report auhotrs can apply [sensitivity labels](../admin/service-security-data-protection-overview.md) to reports, upon export to Excel, PowerPoint or PDF file Power BI will apply report's sensitivity label and label's protection settings on the exported file.
```
Comment (mihart) [inline powerbi-docs/visuals/power-bi-visualization-export-data.md:169]: Report authors should apply

Comment (antonfrMSFT) [reply powerbi-docs/visuals/power-bi-visualization-export-data.md:169]: Sensitivity label is an opt in optional feature, we can recommend to use it, should might a bit too harsh .

Comment (mihart) [reply powerbi-docs/visuals/power-bi-visualization-export-data.md:169]: to reports. When the report is exported, Power BI will use the sensitivity label to apply the appropriate protection settings to the exported file.  <please add a "for example" here> For example, if the report author has applied the sensitivity label of X, then ABC will happen.

**Comment by PRMerger20** [comment]
@antonfrMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@mihart : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**